sdk/src/main/java/com/mapbox/maps/renderer/FpsManager.kt

```diff
@@ -0,0 +1,180 @@
+package com.mapbox.maps.renderer
+
+import android.os.Handler
+import android.os.Looper
+import androidx.annotation.AnyThread
+import androidx.annotation.WorkerThread
+import androidx.core.os.postDelayed
+import com.mapbox.maps.BuildConfig
+import com.mapbox.maps.logI
+import com.mapbox.maps.logW
+import kotlin.math.pow
+
+internal class FpsManager {
```

Comment:
nit: add JvmSynthetic annotation

Reply:
why do we need it for an internal class?

Reply:
got it, thanks, will do

Reply:
@pengdev `@ JvmSynthetic` could not be actually applied to whole class, any ideas how you see it should be applied then?

Reply:
discussed offline, should be treated separately as setting `@file:JvmSynthetic` in the beginning of the file actually does not matter for API checks (for some reason, needs investigation)